R/register_extendr.R

```diff
@@ -15,18 +15,22 @@
 #' @param force_wrappers Logical indicating whether to generate a minimal
 #'   wrapper in the cases when the package's namespace cannot be loaded. This is
 #'   useful to recover the wrapper file when something is wrong with it.
-#' @return The generated wrapper code. Note that this is not normally needed,
-#' as the function saves the wrapper code to `R/extendr-wrappers.R`.
+#' @return Logical scalar. `TRUE` if new wrappers were emitted, `FALSE`
+#'   if wrappers did not require an update.
```

Comment:
Isn't it more straightforward to show some message like "Skip generating wrappers"?

Reply:
My idea is that because it is an exported function that *may* be called separately at some point, it will provide information about the package state: `FALSE` if things are up to date, `TRUE` if new wrappers were generated, and exception if things are bad.

Reply:
Okay, makes sense. Then, how about returning the paths to the updated files (i.e. `R/extendr-wrappers.R`)? That's consistent with what the equivalents of cpp11 and Rcpp do:

- https://cpp11.r-lib.org/reference/cpp_register.html#value
- https://github.com/RcppCore/Rcpp/blob/c5a97a3092bb038dc4236133144ebdb877764a72/man/compileAttributes.Rd#L29

Reply:
Done.